sdk/src/main/java/io/opentelemetry/sdk/trace/Samplers.java

```diff
@@ -89,6 +90,8 @@ public Decision shouldSample(
         TraceId traceId,
         SpanId spanId,
         String name,
+        Span.Kind spanKind,
+        Collection<AttributeValue> attributes,
```

Comment:
We need the key, value here, don't we?

Reply:
oh yeah! Good catch. Yes, this needs to be the Map.

Reply:
Oh my bad! I understood the specification to have access only to the values of the attributes.